### PR TITLE
Add more porting providers to Console support in Godot

### PR DESCRIPTION
**Pull request by nightblade9**
Add two more publishers. Sorted alphabetically.
Disclosure: I do not work for or have used either of these publishers myself.

### CODE REVIEW

File: tutorials/platform/consoles.rst [hunk 1]
```diff
@@ -62,11 +62,16 @@ your games to various consoles.
...
+- `Deck13 Interactive <https://www.deck13.com/>`_ offers Switch porting
+  and publishing of Godot games.
 - `Lone Wolf Technology <http://www.lonewolftechnology.com/>`_ offers
   Switch and PS4 porting and publishing of Godot games.
 - `Pineapple Works <https://pineapple.works/>`_ offers
   Switch and Xbox One porting and publishing of Godot games.
-
+- `Ratalaika Games <https://www.ratalaikagames.com//>`_ offers
+  Switch, Xbox One, XBox Series X, Playstation 4, and Playstation 5 porting
+  and publishing of Godot games.
```
Comment (Calinou) [inline tutorials/platform/consoles.rst:73]:
```suggestion
- `Ratalaika Games <https://www.ratalaikagames.com//>`_ offers
  Switch, Xbox One, Xbox Series, PlayStation 4, and PlayStation 5 porting
  and publishing of Godot games.
```

File: tutorials/platform/consoles.rst [hunk 2]
```diff
@@ -62,11 +62,16 @@ your games to various consoles.
 
 Following is the list of providers:
 
+- `Deck13 Interactive <https://www.deck13.com/>`_ offers Switch porting
+  and publishing of Godot games.
 - `Lone Wolf Technology <http://www.lonewolftechnology.com/>`_ offers
   Switch and PS4 porting and publishing of Godot games.
```
Comment (Calinou) [inline tutorials/platform/consoles.rst:68]:
```suggestion
- `Lone Wolf Technology <http://www.lonewolftechnology.com/>`_ offers
  Switch and PlayStation 4 porting and publishing of Godot games.
```

File: tutorials/platform/consoles.rst [hunk 3]
```diff
@@ -62,11 +62,16 @@ your games to various consoles.
...
 
+- `Deck13 Interactive <https://www.deck13.com/>`_ offers Switch porting
+  and publishing of Godot games.
 - `Lone Wolf Technology <http://www.lonewolftechnology.com/>`_ offers
-  Switch and PS4 porting and publishing of Godot games.
+  Switch and PlayStation 4 porting and publishing of Godot games.
 - `Pineapple Works <https://pineapple.works/>`_ offers
   Switch and Xbox One porting and publishing of Godot games.
-
+- `Ratalaika Games <https://www.ratalaikagames.com//>`_ offers
```
Comment (Calinou) [inline tutorials/platform/consoles.rst:71]:
```suggestion
- `Ratalaika Games <https://www.ratalaikagames.com/>`_ offers
```

**Comment by nightblade9** [comment]
Thanks for the suggestions. It looks like you already resolved them. Is there any action pending on my side?

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by wkubiak** [comment]
Has this new information been somehow checked/verified? @nightblade9 @Calinou 

IIRC. Deck13 published Resolutiion, but do they actually also directly handle porting now?
Resolutiion itself was ported by the game's developers using Lone Wolf's tech (it's public info, statements from the developers can be easily found on twitter and reddit).

Ratalaika = any official news about console porting from Godot?

**Comment by akien-mga** [comment]
Yeah that doesn't seem accurate to me.

AFAIK Ratalaika was indeed working on a Godot Switch port, but I haven't seen any title actually published nor a confirmation that they do provide Godot porting now. https://twitter.com/ratalaika/status/1339623563364626433?lang=en
I definitely did not see any info about the other platforms listed in this PR, Ratalaika does port to those platforms but not necessarily Godot games (there's no single hit for "godot" on their website).

**Comment by nightblade9** [comment]
Thank you for challenging this, I was actually expecting some scrutiny 🙂 I checked/verified everything before updating. Here's what I used:

- Ratalaika games: [this tweet](https://twitter.com/RatalaikaGames/status/1442873340775976967) where their official Twitter account mentions that they port to Godot.
- For Deck13, it was probably a combination of Resolutiion (I think that's how I discovered them), and the CrossCode Steam page mentioning them as a publisher. But, in hindsight, perhaps I'm conflating publishing and porting.

If you would like to roll this back pending further verification, that works too. Is there some sort of "official" requirement/rule/guideline/something before companies can be listed here?

**Comment by pycbouh** [comment]
> Is there some sort of "official" requirement/rule/guideline/something before companies can be listed here?

Well, this list doesn't aim to be exhaustive, but as a good rule of thumb for any list of third parties that we show – it should be confirmed/proposed by the party itself, or their affiliation is certain from the public record. I.e. don't base inclusion on a guess so that we don't mislead users.

**Comment by wkubiak** [comment]
@nightblade9 if you read that twitter thread very closely, Ratalaika and Penguin Pop Games actually don't confirm that they are currently able to do console porting from Godot. I think it's hard to define a "yeah someday maybe" answer as sufficient proof in this context.

Deck13 is a games publisher and they have signed up at least two Godot games to date (Resolutiion and Spindle). This isn't directly related to providing porting capabilities though. I mean, there's now more publishers who have signed up Godot games and have console version plans for those titles so by this logic every publisher like that could be included.

I'm all for rolling back this merge/changes to the page as the new unconfirmed entries are simply misleading.

**Pull request merged**